### DISCUSSION TITLE
rcssmin and rjsmin dependencies

### DISCUSSION THREAD

**Discussion by mdesco18** [django-compressor/django-compressor#1065] [Q&A]
When installing _django-compressor_, is it necessary to also install _rcssmin_ and _rjsmin_?

Such as if I were to install a list of dependencies using `pip install -r requirements.txt`
where _requirements.txt_ contains:

> django-compressor==2.3
> rcssmin==1.0.6
> rjsmin==1.1.0

are rcssmin and rjsmin redundant?

Thanks in advance,

**Comment by jaap3** [comment]
django-compressor pins both these packages in [`setup.py`](https://github.com/django-compressor/django-compressor/blob/022e362b55f6436ac01c820d65b7be8566bc50ba/setup.py#L149). So it's not strictly necessary to include these packages in `requirements.txt` as they will be installed as a dependency of django-compressor anyway.

However, the opinions vary on whether you should or should not pin everything you need for a project. But that is another discussion.
